pkg/slack/events/handler.go

```diff
@@ -0,0 +1,93 @@
+package events
+
+import (
+	"github.com/sirupsen/logrus"
+	"github.com/slack-go/slack/slackevents"
+)
+
+// Handler knows how to handle an event callback.
+type Handler interface {
```

Comment:
Whats the point of defining an interface instead of just exporting the concrete types if we don't have tests that implement their version of the interface?

Reply:
The caller should not infer anything from the individual handlers, at the routing level they should be identical and indistinguishable

Reply:
But there is only one implementation or did I miss it?

Reply:
For now, yes. There will be many more as the bot learns how to respond to more events